config/environments/test.rb

```diff
@@ -1,11 +1,13 @@
+# The test environment is used exclusively to run your application's
+# test suite. You never need to work with it otherwise. Remember that
+# your test database is "scratch space" for the test suite and is wiped
+# and recreated between test runs. Don't rely on the data there!
+
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  # The test environment is used exclusively to run your application's
-  # test suite. You never need to work with it otherwise. Remember that
-  # your test database is "scratch space" for the test suite and is wiped
-  # and recreated between test runs. Don't rely on the data there!
-  config.cache_classes = true
+  config.cache_classes = false
```

Comment:
Why disable cache_classes for a test? I think this can increase performance in tests.

Reply:
It's the new default. Performance difference is negligible. Arguably, `cache_classes = true` is slower:

### with `cache_classes = false`

```
Finished in 1.31 seconds (files took 0.9362 seconds to load)
bundle exec rspec  1,78s user 0,29s system 80% cpu 2,585 total

Finished in 1.25 seconds (files took 0.94156 seconds to load)
bundle exec rspec  1,79s user 0,30s system 81% cpu 2,568 total

Finished in 1.22 seconds (files took 0.87448 seconds to load)
bundle exec rspec  1,76s user 0,28s system 84% cpu 2,412 total

Finished in 1.31 seconds (files took 0.90793 seconds to load)
bundle exec rspec  1,79s user 0,28s system 84% cpu 2,453 total

Finished in 1.23 seconds (files took 0.91102 seconds to load)
bundle exec rspec  1,75s user 0,29s system 81% cpu 2,503 total
```

#### avg

(1.31+1.25+1.22+1.31+1.23)/5=1.264
(1.78+1.79+1.76+1.79+1.75)/5=1.774

### with `cache_classes = true`

```
Finished in 1.42 seconds (files took 0.96655 seconds to load)
bundle exec rspec  1,93s user 0,30s system 84% cpu 2,651 total

Finished in 1.26 seconds (files took 0.72145 seconds to load)
bundle exec rspec  1,78s user 0,26s system 91% cpu 2,238 total

Finished in 1.36 seconds (files took 0.71262 seconds to load)
bundle exec rspec  1,87s user 0,27s system 91% cpu 2,326 total

Finished in 1.27 seconds (files took 0.70377 seconds to load)
bundle exec rspec  1,77s user 0,25s system 91% cpu 2,212 total

Finished in 1.24 seconds (files took 0.69918 seconds to load)
bundle exec rspec  1,78s user 0,26s system 91% cpu 2,218 total
```

#### avg

(1.42+1.26+1.36+1.27+1.24)/5=1.31
(1.93+1.78+1.87+1.77+1.78)/5=1.826